web-starter/templates/_.gitignore

```diff
@@ -0,0 +1,21 @@
+# Ignore directories generated by Composer
+/drush/contrib/
+/vendor/
+/<%= services.web.doc_root %>/core/
+/<%= services.web.doc_root %>/modules/contrib/
+/<%= services.web.doc_root %>/themes/contrib/
+/<%= services.web.doc_root %>/profiles/contrib/
+/<%= services.web.doc_root %>/libraries/
+
+# Ignore sensitive information
+/<%= services.web.doc_root %>/sites/*/settings.php
+/<%= services.web.doc_root %>/sites/*/settings.local.php
+
+# Ignore Drupal's file directory
+/<%= services.web.doc_root %>/sites/*/files/
+
+# Ignore SimpleTest multi-site environment.
+/<%= services.web.doc_root %>/sites/simpletest
+
+# Ignore files generated by PhpStorm
+/.idea/
```

Comment:
Might want to remove this line -- in my opinion, things like this should be handled by the gitignore in the app generator

Reply:
There are also now facilities to update the .gitignore file directly in the main generator. See https://github.com/forumone/generator-web-starter/commit/c5a92f2ceab86607d13a18c8660716027ef745d9

Reply:
I'm using that -- see https://github.com/forumone/generator-web-starter-drupal8/pull/8/files#diff-f09ef04ace4653c8cb420d336d41a523R102